### ISSUE TITLE
[v2 API] Path based FS examples

### ISSUE THREAD

**Issue by zakkor** [hanwen/go-fuse#346]
With the old API, I can see methods in the pathfs package that make creating a path-based FS very easy - is this possible with the new API? The readme seems to indicate yes:

> Nodes contain references to their parents. As a result, we can derive the path for each Inode, and there is no need for a separate PathFS.

But I can't see how that's relevant - perhaps some examples would make it easier to understand?

**Comment by hanwen** [comment]
good point. The Path() method on Node should give you enough to implement path based system.

**Comment by muryoutaisuu** [comment]
I second this. I wanted to update the libraries in my project (which heavily relied on pathfs package in addition to nodefs) and just realized now, it's being discontinued. I'll dig into your mentioned Path() method, then I'll see, how easy it's to migrate to the new API without an example.
